packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -14,9 +14,14 @@ jest.mock('@salesforce/core', () => {
 describe('test project paths', () => {
   const hasRootWorkspaceStub = jest.spyOn(workspaceUtils, 'hasRootWorkspace');
   const FAKE_WORKSPACE = '/here/is/a/fake/path/to/';
+  const FAKE_WORKSPACE_INSTANCE: any = {};
   const FAKE_STATE_FOLDER = '.sfdx';
   const FAKE_CONFIG_FILE = 'sfdx-config.json';
+  const FAKE_USERNAME = 'testuser';
+  const FAKE_METADATA = 'metadata';
+  const FAKE_ORGS = 'orgs';
```

Comment:
We should export these values in paths.ts and re-use them here.  It doesn't make sense to create the constant in both places.

Reply:
Have exported METADATA, ORGS and other constants in paths.ts, which are reused in the test file.